src/app/globals.css

```
@@ -1,26 +1,323 @@
-@media (prefers-color-scheme: dark) {
-  :root {
-    --background: #0a0a0a;
-    --foreground: #ededed;
+/* KCVV Logo Spinner Animation (matches Gatsby site) */
+@keyframes kcvv-logo-spin {
+  0%,
+  100% {
+    animation-timing-function: cubic-bezier(0.5, 0, 1, 0.5);
+  }
+
+  0% {
```

Biome (2.1.2)

[error] The duplicate keyframe selector is overwritten by later one.

Consider using a different percentage value or keyword to avoid duplication

(lint/suspicious/noDuplicateSelectorsKeyframeBlock)